ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ModifyVulnReport/CloneVulnReportPage.tsx

```diff
@@ -153,22 +107,14 @@ function CloneVulnReportPage() {
             </PageSection>
             <Divider component="div" />
             <PageSection padding={{ default: 'noPadding' }} isCenterAligned>
-                <Wizard
+                <ReportFormWizard
+                    formik={formik}
                     navAriaLabel="Report clone steps"
                     mainAriaLabel="Report clone content"
-                    hasNoBodyPadding
-                    steps={wizardSteps}
+                    wizardStepNames={wizardStepNames}
+                    finalStepNextButtonText={'Create'}
```

Comment:
As a prerequisite for follow up of main form upgrade, can we ask in UI/UX meeting whether there is any PatternFly guideline to render more specific text than **Save** for clone/create versus edit actions?

Reply:
Many other forms consistently render **Save** and I think the team has discussed the issue before.

Reply:
Aha, similar for `wizardStepNames` which distinguish **Review and create** from **Review and save** when I see only **Review** in a quick scroll through `Wizard` design guidelines:
https://www.patternfly.org/components/wizard/design-guidelines